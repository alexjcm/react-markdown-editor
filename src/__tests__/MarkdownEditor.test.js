import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';

import MardownEditor from '../components/MarkdownEditor';

/**
 * This test mounts a component and makes sure that it didnâ€™t throw during rendering.
 */
test('renders without crashing', () => {
  shallow(<MardownEditor />);
});

/**
 * test if the UI renders correctly
 */
test('renders correctly', () => {
  const rendered = renderer.create(<MardownEditor />);
  expect(rendered.toJSON()).toMatchSnapshot();
});
