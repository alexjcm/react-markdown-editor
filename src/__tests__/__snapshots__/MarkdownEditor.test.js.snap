// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <div
    className="contain"
  >
    <div
      className="headerMD"
    >
      MARKDOWN
    </div>
    <textarea
      autoFocus={true}
      className="textarea"
      onChange={[Function]}
    />
  </div>,
  <div
    className="contain"
  >
    <div
      className="headerMD"
    >
      PREVIEW
    </div>
    <div
      className="markdown"
    />
  </div>,
]
`;
